### DISCUSSION TITLE
How to start a phase field fracture analysis?

### DISCUSSION THREAD

**Discussion by rh201** [idaholab/moose#17169] [Q&A Modules: Phase field] [answered]
Hello all,

I'm new to MOOSE and want to do phase field fracture analysis, but I have no idea how to start and what should I do for now. I was wondering if there are any simple examples or tutorials to shed light on me. Any advice or comments will be highly appreciated.

Thanks,
RH

**Comment by cticenhour** [comment, marked as answer]
If you are looking for help specific to the phase field module, the main documentation page for that module might be a good place to start: https://mooseframework.inl.gov/modules/phase_field/index.html

If you want to work through a generic MOOSE tutorial and see other "getting started" resources, then I would go here: https://mooseframework.inl.gov/getting_started/examples_and_tutorials/index.html

**Reply by rh201** [reply]
Thanks @cticenhour .

I've learned the basic parts of MOOSE already, including the page you linked,and now I want to proceed to phase field fracture, wondering if there are any related examples.

**Reply by cticenhour** [reply]
Excellent - just wanted to make sure you had seen these since you said you were new to the framework! I'll let someone with experience in phase field continue the conversation. Perhaps @laagesen can chime in or recommend other resources?

**Reply by laagesen** [reply]
Probably @jiangwen84 is the best person to highlight some PF fracture examples

**Reply by jiangwen84** [reply]
We have a few regression tests located at moose/modules/combined/test/tests/phase_field_fracture/ 

The document of phase-field fracture is at https://mooseframework.inl.gov/source/materials/ComputeLinearElasticPFFractureStress.html

**Reply by rh201** [reply]
Thank you.
